src/components/Savings/SavingsModal/index.tsx

```diff
@@ -103,8 +103,8 @@ const SavingsModal = memo(
         useEffect(() => {
```

Comment:
I see this effect just does calculation. if setBalance / setWithdrawAmount aren't used anywhere else, could be replaced with useMemo

Reply:
replaced